src/hooks/tests/use-import-export.tsx

```diff
@@ -43,7 +43,9 @@ describe( 'useImportExport hook', () => {
 		expect( result.current.exportState ).toEqual( {} );
 		expect( getIpcApi().exportSite ).toHaveBeenCalledWith(
 			{
-				sitePath: '/test-site',
+				site: {
+					...selectedSite,
+				},
```

Comment:
Unless we expect to modify it, I think we don't need to spread `selectedSite`. We could do:
```suggestion
				site: selectedSite,
```